doc/api/esm.md

```diff
@@ -95,7 +95,7 @@ until the root of the volume is reached.
 ```
 
 ```bash
-# In same folder as above package.json
+# In same folder as previous package.json
```

Comment:
> It also applies within a section because a screenreader would still say "above" which doesn't make much sense without the visual appearance of blocks of written text. :)

But surely users of screenreading software know that English is written left-to-right, top-to-bottom.

@Trott sure, “preceding” is fine. It doesn’t make me think I need to start scrolling way up. I still feel like there should be an even better word though, that means “this one, right here, the thing that’s part of this section.” Like for me, there’s a difference between referring to the various parts of the current example, and referring to something outside of it.